apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -5297,8 +5297,8 @@
       "description": "Please use My Ledger to uninstall all apps then check if a firmware update is available before reinstalling them"
     },
     "LatestFirmwareVersionRequired": {
-      "title": "Update Nano Firmware",
-      "description": "Please update your devices Firmware to the latest version to access this feature"
+      "title": "Update Device Firmware",
+      "description": "Please update your devices firmware to the latest version to access this feature"
```

Comment:
Shouldn't we also change the text here so it reads "Please update your device firmware to the latest version to access this feature" (device singular since user is only updating one device)?

Reply:
I will do an update when the decision is made 👍

Reply:
Agree with Allane :)

Reply:
👌